### PR TITLE
Chronos/Nano: unset the KMP_INIT_AT_FORK for pytorch training

### PR DESCRIPTION
**Pull request by TheaperDeng**
related to
https://github.com/intel-analytics/BigDL/issues/4689
https://github.com/intel-analytics/BigDL/issues/4590

### CODE REVIEW

File: python/chronos/src/bigdl/chronos/forecaster/__init__.py [hunk 1]
```diff
@@ -13,7 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 import warnings
+# unset the KMP_INIT_AT_FORK
+# which will cause significant slow down in multiprocessing training
+import os
+os.unsetenv('KMP_INIT_AT_FORK')
```
Comment (jason-dai) [inline python/chronos/src/bigdl/chronos/forecaster/__init__.py:21]: why do you need to unset it in both nano and chronos?

Comment (TheaperDeng) [reply python/chronos/src/bigdl/chronos/forecaster/__init__.py:21]: We should avoid any `import torch` or with this env variable still exist.
In Chronos, nano trainer is imported really late (during `fit`), torch is imported before nano trainer's import, so I need to do it here.

**Comment by TheaperDeng** [comment]
@yangw1234 any comments on this PR?

**Comment by yangw1234** [comment]
LGTM

**Comment by TheaperDeng** [comment]
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/225/
http://10.112.231.51:18888/job/BigDL-PRVN-chronos-Python-Spark-3.1-py37-Horovod/1106/

**Pull request merged**